script/check-aws-cdk

```diff
@@ -11,7 +11,7 @@ VERSIONS=$(npm view $LIBRARY_TO_CHECK versions --json)
 
 TOTAL_VERSIONS=$(echo "$VERSIONS" | jq '. | length')
 
-INSTALLED_VERSION_NUMBER=$(jq -r ".devDependencies.\"${LIBRARY_TO_CHECK}\"" < "$DIR/../package.json")
+INSTALLED_VERSION_NUMBER=$(jq -r ".dependencies.\"${LIBRARY_TO_CHECK}\"" < "$DIR/../package.json")
```

Comment:
We [want to](https://github.com/guardian/cdk/pull/1174#discussion_r847205323) remove this script entirely as Dependabot should work for us now. Let's do that soon!